templates/pgbackrest.conf.j2

```diff
@@ -1,4 +1,6 @@
 [global]
+lock-path=/var/snap/charmed-postgresql/common/locks
+log-path=/var/snap/charmed-postgresql/common/logs
```

Comment:
New paths needed by pgBackRest (that replace the default paths from outside the snap paths).